ui/packages/consul-ui/app/components/hashicorp-consul/index.hbs

```diff
@@ -388,6 +388,9 @@
     </:main>
 
     <:content-info>
+      <p>
+        Consul v{{env 'CONSUL_VERSION'}}
+      </p>
```

Comment:
Here's a link to what this file/bunch of lines used to look like before we took the version our in 1.10.x

https://github.com/hashicorp/consul/blob/52dced80329b9f885707e6c57b85001fc829ae62/ui/packages/consul-ui/app/components/hashicorp-consul/index.hbs#L253